script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -106,8 +106,9 @@ The human vessel cannot withstand such attrition,
 [sha:226b0316df7656509b12705329931abdaf56beb1]{
 -- Page 5, Offset 37023.
 -- 吸血種として不死の肉体を手にいれても、精神の鈍化は止められない。
-Even with the immortal body of a bloodsucker,
- one still wouldn't be able to prevent their mind from growing dull.
+// Trying to keep the next line on the page
+Even with the immortality of a bloodsucker,
+ one wouldn't be able to prevent their mind from growing dull.
```

Comment:
Ok now this one I _had_ already fixed. 

![2023-04-09-220521_2160x1286_scrot](https://user-images.githubusercontent.com/2476799/230811038-e87cc235-9ff0-4b86-9433-3bbd67800edd.png)